src/functions/utils.php

```diff
@@ -158,6 +158,30 @@ function tribe_get_request_var( $var, $default = null ) {
 	}
 }
 
+if ( ! function_exists( 'tribe_get_global_query_object' ) ) {
+	/**
+	 * Grabs the $wp_query global in a safe way with some fallbacks that help prevent fatal errors
+	 * on sites where themes or other plugins directly manipulate the $wp_query global.
+	 *
+	 * @since TBD
+	 *
+	 * @return object|boolean The $wp_query, the $wp_the_query if $wp_query empty, false otherwise.
+	 */
+	function tribe_get_global_query_object() {
+		global $wp_query;
```

Comment:
I don't have a strong feeling either way but since it caught my eye ... why not also import the `$wp_the_query` global, instead of mixing and matching styles (for one global you're doing it this way, for the other you're pulling from the `$GLOBALS` array)?